src/AppBridgeBlock.ts

```diff
@@ -445,7 +440,7 @@ export class AppBridgeBlock {
         };
     }
 
-    private mapTemplateApiToFrontifyTemplate(template: TemplateApi): FrontifyTemplate {
+    private mapTemplateApiToFrontifyTemplate(template: TemplateApi): Template {
```

Comment:
We should update the method name as well:
```suggestion
    private mapTemplateApiToTemplate(template: TemplateApi): Template {
```